earth_enterprise/legacy_installer/README.txt

```diff
@@ -0,0 +1 @@
+This is the obsolete installer. See ../src/installer for the scripts you want.
```

Comment:
Some scripts from earth_enterprise/installer are still being used. For example, when you run
scons -j8 release=1 install install_dir... on the level of earth_enterprise, it runs installer/Scripts/verify_installerdir.sh
Please look for installer/Scripts/verify_installerdir.sh in the earth_enterprise/SConstruct.

So, maybe do not move files do far, but put in readme of earth_enterprise/installer, that \
"This is legacy installer used for continuous build with pulse and InstallAnywhere packager. 
Some of the script (e.g. installer/Scripts/verify_installerdir.sh) is still being used in the new install-scripts.
TODO: clean up ...